setup.py

```diff
@@ -12,9 +15,36 @@
     read_embree_location, \
     in_conda_env
 from distutils.version import LooseVersion
+from distutils.ccompiler import CCompiler
 import pkg_resources
 
 
+def _get_cpu_count():
+    if platform.system() != "Windows":
+        return os.cpu_count()
```

Comment:
I'm curious what happens if somebody does run this with a Windows machine? Do we handle that somewhere else?

Reply:
This function will return `None`, which is a default value for `nthreads` in `cythonize` and it will evaluate to `False` in if statement in `build_extensions`

Reply:
Great! Thanks for explaining that. 🙂

Reply:
I just noticed it's actually 0 that's default but it looks like it works with `None` too. However, I'll fix that